localization/english/dvg_flavor_ottomans_l_english.yml

```diff
@@ -29,3 +29,6 @@
   dvg_ottomans_reclaim.4.t:0 "The Serbs Pacified"
   dvg_ottomans_reclaim.4.d:0 "Despite all attempts, the Serbs have been drowned by the Turkish tide, and have no choice but to re-enter into the Empire."
   dvg_ottomans_reclaim.4.a:0 "The rebellious son returns."
+  #macedonian decisions, this is very placeholder, until the balkan rework is done.
+  dvg_mcd_acceptgreeks:0"Accept Greeks"
```

Comment:
![image](https://github.com/vic3-divergences/divergences/assets/116951759/5733dfce-8b3d-400e-92c6-ac0799deba7a)
Not showing the localization

[11:36:37][localization_reader.cpp:536]: Missing quoted string value for key 'dvg_mcd_acceptgreeks' at line 33 and column 23 in localization/english/dvg_flavor_ottomans_l_english.yml
[11:36:37][localization_reader.cpp:536]: Missing quoted string value for key 'dvg_mcd_acceptgreeks_desc' at line 34 and column 28 in localization/english/dvg_flavor_ottomans_l_english.yml